crates/bevy_ecs/src/schedule/mod.rs

```diff
@@ -1,3 +1,7 @@
+//! Tools for controlling system execution.
```

Comment:
```suggestion
//! Tools for controlling system execution order.
```

Reply:
Hmm, I'm not 100% sold on this one. While it's true that the `schedule` module contains tools to execute systems in a certain order (`SystemStage`), it is much more: there are executors and run criteria for example.